### PR TITLE
KOGITO-5642: Do not run github action for some paths

### PR DESCRIPTION
**Pull request by jomarko**

### CODE REVIEW

File: .github/workflows/sonar_analysis_kiegroup.yml [hunk 1]
```diff
@@ -3,6 +3,14 @@ on:
   push:
     branches:
       - main
+    paths-ignore:
```
Comment (radtriste) [inline .github/workflows/sonar_analysis_kiegroup.yml:6]: I guess this could be added also to other workflows or ?

Comment (jomarko) [reply .github/workflows/sonar_analysis_kiegroup.yml:6]: @radtriste thank you, I tried to add the ignore-paths on places I think is applicable

**Review by radtriste — APPROVED** [review]
lgtm

**Comment by jomarko** [comment]
Failing FBD is a different issue - #110

@kiegroup/gatekeepers may I ask for a merge? Sure, if some engineer available for review, review is welcome before merge too.

**Pull request merged**